### DISCUSSION TITLE
Code Performance Ranking

### DISCUSSION THREAD

**Discussion by ssnyder-intrinio** [codewars/codewars.com#2553] [Ideas]
I see that you clock the run time for an answer. Can we get a performance ranking for kata answers seperate from community voted clever and best-practice? (This stems from seeing multiple katas voted highly, yet the implementation is very bad performance-wise).

Naturally, this would need multiple data points of run times at differing lengths to determine it's performance and scaling.
ex) Short inputs, medium inputs, and long inputs. One answer could own the short input best performance, another (or same) could own the medium input best performance, etc.

**Comment by hobovsky** [comment]
Please see #959 (and other, linked issues) for some discussion on related topic.
